### DISCUSSION TITLE
欢迎来到Autovisor讨论版块！大家可以交流问题和提出改进建议～

### DISCUSSION THREAD

**Discussion by CXRunfree** [CXRunfree/Autovisor#8] [Announcements]
Feel free to chat here.

**Comment by Rremilia111** [comment]
老哥，知到有时候会只播放个位数% 然后跳到下一个视频 这是bug还是操作不当？

**Reply by Rremilia111** [reply]
![image](https://github.com/CXRunfree/Autovisor/assets/165046358/007afb35-8f73-449d-98bf-7b7de8ebffec)

**Comment by CXRunfree** [comment]
旧版本可能有各种不稳定bug，建议更新至新版
